GitHubContributionsWidget/es.lproj/GitHubContributionsWidget.strings

```diff
@@ -24,6 +28,10 @@
 
 "O7wabU" = "Morado";
 
+"ub2khZ" = "Amarillo";
+
+"ywLI12" = "Inversión";
```

Comment:
I would maybe translate it to "Invertido" in spanish 🤔

Reply:
fixed at https://github.com/AnderGoig/github-contributions-ios/pull/20/commits/64cd790b0406ce2c37c65d610546c3d0ce1ae2ce